src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -735,4 +735,10 @@ Parameter name: {0}</value>
     <value>The current user, '{0}', is not a maintainer of the package '{1}'.</value>
     <comment>{0} = user name (i.e. 'DynamoTeam'), {1} = package name (i.e. 'Clockwork for Dynamo 1.x')</comment>
   </data>
+  <data name="DuplicatedOlderPackage" xml:space="preserve">
+    <value>An older version of the package called {0} version {2} was found at {1} with version {3}. Ignoring it.</value>
```

Comment:
`Ignoring it.` is kind of unclear to me...Which one is ignored...and I guess "to ignore" in this context is something obvious to the user...

Reply:
What about changing `Ignoring it.` to `The older version has been ignored.`?

Reply:
I think that is better ...